model_analyzer/config/generate/coordinate.py

```diff
@@ -74,6 +74,12 @@ def __eq__(self, other: Any) -> bool:
                 return False
         return True
 
+    def __lt__(self, other: Any) -> bool:
```

Comment:
Doesn't other have to be a List[int] here (and same for the above dunder methods)?

Reply:
this code is gone